### DISCUSSION TITLE
How to use translate

### DISCUSSION THREAD

**Discussion by GuilhermeLegramante** [Power-Components/livewire-powergrid#363] [Q&A] [answered]
How can I set translate to pt_BR?

**Comment by dansysanalyst** [comment, marked as answer]
Hi @GuilhermeLegramante, 

PowerGrid language will follow the application `locale` set at `config/app.php`.

```php

//...
    /*
    |--------------------------------------------------------------------------
    | Application Locale Configuration
    |--------------------------------------------------------------------------
    |
    | The application locale determines the default locale that will be used
    | by the translation service provider. You are free to set this value
    | to any of the locales which will be supported by the application.
    |
    */

    'locale' => 'pt_br',
```

Results in:

<img width="1156" alt="CleanShot 2022-04-05 at 15 22 35@2x" src="https://user-images.githubusercontent.com/79267265/161763553-a9fb7e3e-77c3-4894-8913-2f0373b622f4.png">



________
⭐ Enjoying PowerGrid? Please consider Starring the repository!
📢 Share PowerGrid with your dev-community, friends and colleagues: Tweet, write a blog, record review video. 
👥 PowerGrid is open-source, entirely developed and maintained by the author and the contributors.

**Reply by GuilhermeLegramante** [reply]
Thanks!
